core-bundle/src/Resources/contao/classes/FrontendUser.php

```diff
@@ -122,7 +122,7 @@ public function authenticate()
 		if (\Config::get('autologin') > 0 && ($strCookie = \Input::cookie('FE_AUTO_LOGIN')))
 		{
 			// Try to find the user by his auto login cookie
-			if ($this->findBy('autologin', $strCookie) !== false)
+			if ($this->findBy('autologin', hash_hmac('sha256', $strCookie, \Config::get('encryptionKey'))) !== false)
```

Comment:
Should we use `\System::getContainer()->getParameter('kernel.secret')` or `\System::getContainer()->getParameter('contao.encryption_key')` here instead?

Reply:
Yes. I'll adjust the PR in a minute.